src/app/views/album-detail.html

```diff
@@ -1,14 +1,44 @@
 <div>
-    <h2>EN ESTA SECCION TIENE QUE APARECER LA TAPA DEL DISCO SELECCIONADO JUNTO CON LA LISTA DE TEMAS QUE TIENE EL DISCO.</h2>
-    <img src="https://i.scdn.co/image/66302ca80395b6be3600d5f0ef69db9e0c43f4f5" alt="">
-    <ul>
-        <li>Track 1</li>
-        <li>Track 2</li>
-        <li>Track 3</li>
-        <li>Track 4</li>
-        <li>Track 5</li>
-        <li>Track 6</li>
-        <li>Track 7</li>
-        <li>Track 8</li>
-    </ul>
+    <nav class="nav">
+        <img src="/app/images/spotify-icon.png" alt="" class="logo-left" ng-click=albumDetailCtrl.toIndex()>
+        <div class="column-search">
+            <input type="text" class="search-input" placeholder="Search for another artist" ng-model="albumDetailCtrl.search">
+            <i class="fa fa-search" aria-hidden="true" id="search-icon" ng-click="albumDetailCtrl.toResults()"></i>
```

Comment:
This is wrong. You only add ng-click handlers only to `buttons`, adding it to any other element is a hacky way. Please wrap `i` elem with a button and add ng-click to the button elem. It would be great if you can send the user to the same place by pressing Intro key inside the input elem.